gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -791,6 +791,12 @@ gst_tensor_filter_fixate_caps (GstBaseTransform * trans,
   silent_debug_caps (caps, "caps");
   silent_debug_caps (othercaps, "othercaps");
 
+  /** Removes no-used-variable warning for priv in when DBG is set */
+  if (priv->fw == NULL) {
+    gst_caps_unref (othercaps);
```

Comment:
Are we safe to do unref othercaps here?
(I now feels that the unref-othercaps at line 805 as well; othercaps is given by the caller and we are unref-ing in the function.)